### ISSUE TITLE
Languagetool shortcut not working

### ISSUE THREAD

**Issue by PMassicotte** [languagetool-org/languagetool#7099]
I am trying to use the newly added shortcut capability, but I can not get it working. I am using version 5.4.3 on Ubuntu with Firefox 104.02.

I have enabled it:

![image](https://user-images.githubusercontent.com/4519221/190406527-16d781e4-808f-45d1-a8f8-66eeea98ad1f.png)

The shortcut is set on `ctrl`. Even if I fast double tap `ctrl`, nothing happens.

![image](https://user-images.githubusercontent.com/4519221/190407122-7c506499-218d-4280-8b34-c2d4eb10b5cf.png)

Any ideas?

**Comment by danielnaber** [comment]
I have the same issue. It's not 100% reproducible, but it feels like it's not working 90% of the time. I'm also using Firefox and Ubuntu, maybe it's only this combination.

**Comment by PMassicotte** [comment]
Yes, I tried all key modifiers. I just tried on Chromium and nothing happens also.

**Comment by PMassicotte** [comment]
Yes, I tried all key modifiers. I just tried on Chromium and nothing happens also.

**Comment by danielnaber** [comment]
> I just tried on Chromium and nothing happens also.

For me, the shortcut works on Chromium.

**Comment by PMassicotte** [comment]
Just got it working on Chrome. Still no luck on Firefox.

**Comment by tiff** [comment]
Thanks for checking

**Comment by danielnaber** [comment]
I've also asked in the Firefox add-on forum. If we have an update, we should post it there, too: https://discourse.mozilla.org/t/problem-with-custom-shortcut-and-linux-ubuntu/105647

**Comment by danielnaber** [comment]
Just today I noticed that I can usually make the error card open if I just keep pressing the shortcut often enough. Sometimes it takes 3 key presses, sometimes it takes 8 or more.

**Comment by danielnaber** [comment]
FYI, I still have the issue with Firefox, while today with Thunderbird 102 I could not reproduce the problem. But we also got a report today from someone using Ubuntu and he couldn't make it work in Firefox nor Thunderbird.

**Comment by Beurt** [comment]
It is maybe useful for you. I have the bug (I use KDE Neon, which is based on Ubuntu LTS).

On this page: https://www.jeffersonscher.com/res/keyvents.html : 2×<kbd>Ctrl</kbd>, 2×<kbd>Alt</kbd>, 2×<kbd>Shift</kbd> does NOTHING, but 2×<kbd>Atl-Gr</kbd> returns results:

> keydown => event.key: "AltGraph"{charCode: 0; keyCode: 225; which: 225; code: AltRight}
> keyup => event.key: "AltGraph"{charCode: 0; keyCode: 225; which: 225; code: AltRight}
>
> keydown => event.key: "AltGraph"{charCode: 0; keyCode: 225; which: 225; code: AltRight}
> keyup => event.key: "AltGraph"{charCode: 0; keyCode: 225; which: 225; code: AltRight}

If Alt-Gr can be added to Firefox Add-on options, it could help!
